src/components/PeopleTable.tsx

```diff
@@ -1,4 +1,78 @@
-export const PeopleTable = () => {
+import { FC } from 'react';
+import { useParams, useSearchParams } from 'react-router-dom';
+import { Person } from '../types';
+import { PersonLink } from './PersonLink';
+import { SortBy } from '../types/SortBy';
+import { SortLink } from './SortLink';
+
+type Props = {
+  people: Person[],
+};
+
+export const PeopleTable: FC<Props> = ({ people }) => {
+  const { slug } = useParams();
+  const [searchParams] = useSearchParams();
+
+  const sex = searchParams.get('sex');
```

Comment:
You can destructure the searchParams object to make the code cleaner. For example, const { sex, centuries, query, sort, order } = searchParams;